tutorials/baseline/auto_trainer.py

```diff
@@ -0,0 +1,111 @@
+import os
+from minio import Minio
+from tqdm.auto import tqdm
+
+import joblib
+import pandas as pd
+
+from typing import Union
+
+import numpy as np
+import scipy.sparse as sp
+import torch
+from torch.utils.data import TensorDataset
+
+from sklearn.feature_extraction.text import ENGLISH_STOP_WORDS
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+from knodle.model.logistic_regression_model import LogisticRegressionModel
+from knodle.trainer.auto_trainer import AutoTrainer
+
+# This python script contains rarely any explanation. For more description, we refer to the corresponding
+# jupyter notebook. There the steps are explained in more detail
+
+
+# Define some functions
+
+def remove_stop_words(text: str) -> str:
+    text = ' '.join([word for word in text.split() if word not in (ENGLISH_STOP_WORDS)])
+    return text
+
+
+def np_to_tensor_dataset(x: np.ndarray):
+    if isinstance(x, sp.csr_matrix):
+        x = x.toarray()
+    x = torch.from_numpy(x)
+    x = TensorDataset(x)
+    return x
+
+
+def create_tfidf_dataset(
```

Comment:
just to be sure: this function will be moved to transforemer.py in later refactoring?

Reply:
yes